### PR TITLE
Closes #55 and #87

### LINKED ISSUE #55: Figure out a better way to sync setup.py version with other version numbers in code

### PR DESCRIPTION
**Pull request by intendo**

### CODE REVIEW

**Review by sobolevnrm — CHANGES_REQUESTED** [review]
I think you need to modify `pdb2pqr/config.py` as well.

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,5 @@
+include README.md
+include setup.py setup.cfg
```
[the remark this answers is not in the document]
Comment (intendo) [reply MANIFEST.in]: Thanks for catching that.

**Comment by sobolevnrm** [comment]
Not sure if you saw the message above but `config.py` needs to be modified as well.

**Comment by intendo** [comment]
Do we just need to remove the VERSION from config.py or make VERSION a template?

**Comment by sobolevnrm** [comment]
Other modules (e.g., `psize` and `inputgen`) and functions `dx_to_cube` in `main` import `VERSION` to set the version information.  Either `config.py` needs to incorporate the information from versioneer so the other code can use it or the other code needs to be updated to pull the information direction from versioneer.

**Pull request merged**